src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -132,6 +132,21 @@ public PublicClientApplicationBuilder WithUseCorporateNetwork(bool useCorporateN
         }
 #endif
 
+#if RUNTIME || NETSTANDARD_BUILDTIME
+        /// <summary>
+        ///  Sets a reference to the ViewController (if using Xamarin.iOS), Activity (if using Xamarin.Android)
+        ///  IWin32Window or IntPtr (if using .Net Framework). Used for invoking the browser.
+        /// </summary>
+        /// <remarks>Mandatory only on Android. Can also be set via the PublicClientApplcation builder.</remarks>
+        /// <param name="parentActivityOrWindowFunc">The parent as an object, so that it can be used from shared NetStandard assemblies</param>
+        /// <returns>The builder to chain the .With methods</returns>
```

Comment:
>/// <returns>The builder to chain the .With methods</returns> [](start = 8, length = 61)

please add the specific android use case using the CurrentActivity (not sure that's the actual name ...)

Reply:
Do we want to have an aka.ms link ? to Interactive with Xamarin?

Reply:
@henrik-me  -- Can you please clarify what you mean about adding the specific android use case?  Do you want another explicit method for .WithActivity() only on Android?  Or are you asking for updates to the XML documentation for this method?

Reply:
@MarkZuber : I don't think we have a specific aka.ms link. Feel free to create one.

Reply:
I believe what Henrik is trying to point at is https://github.com/jamesmontemagno/CurrentActivityPlugin

Reply:
So we want to document this guys' android plugin in our xml documentation for MSAL?

Reply:
Created https://aka.ms/msal-net-android-activity with extra docs around this.  Please review.  @jmprieur  @henrik-me @bgavrilMS

Reply:
I'm asking for updates to the XML doc here. Perhaps that can fit into the wiki page. I recall Bogdan pointing out some of the reasoning behind this method and what people would generally want to do here. I thought it made sense to mention that in our docs. The closer to the developer the better, though some details may fit better in the wiki page.

---
In reply to: [302260651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1261#discussion_r302260651) [](ancestors = 302260651)

Reply:
The guy works for MSFT, though I never quite understood why the plugin is not part of Xamarin, at least Xamarin.Extras

Reply:
aka.ms LGTM!

Reply:
Isn't it also mandatory on iOS when using a broker?

Reply:
@jmprieur yes...it is mandatory on iOS when invoking broker.